app/src/components/fields/CustomTextField.tsx

```diff
@@ -14,6 +14,8 @@ export interface ICustomTextField {
   handleBlur?: FormikContextType<any>['handleBlur'];
 
   maxLength?: number;
+
+  endAdornment?: any;
```

Comment:
Is there a more specific type we can use for this?

Reply:
Reverting this. Unintended change.